webapp/src/tutorialCodeValidation.tsx

```diff
@@ -0,0 +1,61 @@
+/// <reference path="../../built/pxtlib.d.ts" />
+
+import * as React from "react";
+import * as data from "./data";
+import * as sui from "./sui";
+import { TutorialCard } from "./tutorial";
+
+type ISettingsProps = pxt.editor.ISettingsProps;
+
+interface TutorialCodeValidationProps extends ISettingsProps {
+    onYesButtonClick: () => void;
+    onNoButtonClick: () => void;
+    initialVisible: boolean;
+}
+
+interface tutorialCodeValidationState {
+    visible: boolean;
+}
+
+export class MoveOn extends data.Component<TutorialCodeValidationProps, tutorialCodeValidationState> {
+    constructor(props: TutorialCodeValidationProps) {
+        super(props);
+
+        this.state = { visible: this.props.initialVisible };
+    }
+
+    showUnusedBlocksMessage(vis: boolean) {
+        this.setState({ visible: vis });
+    }
+
+    moveOnToNextTutorialStep() {
+        this.props.onYesButtonClick();
+        this.showUnusedBlocksMessage(false);
+    }
+
+    stayOnThisTutorialStep() {
+        this.props.onNoButtonClick();
+    }
+
+    renderCore() {
+        const vis = this.state.visible;
+        return <div>
+            <div className={`tutorialCodeValidation no-select ${!vis ? 'hidden' : ''}`}>
+                <div className="text">
+                    {lf("These are the blocks you seem to be missing.")}
+                </div>
+                <div className="text">
+                    /* Will add a list a blocks here in future PR */
```

Comment:
```suggestion
                    {/* Will add a list a blocks here in future PR */}
```
to avoid having the comment render out to the ui